### ISSUE TITLE
service label without number ?

### ISSUE THREAD

**Issue by patprache** [coreybutler/node-linux#34]
Hi,

I noticed that service with numbers in their name are filtered. For example my program called "pv5web"  becomes "pvweb".

Is there a particular reason? or is it a bug?

**Comment by coreybutler** [comment]
This sounds like a potential bug, but I'm not sure where it is. The names/titles are stripped of special characters on line [434 of systemv.js](https://github.com/coreybutler/node-linux/blob/master/lib/systemv.js#L434), but it should preserve all alphanumeric characters.

I assume this happens when you're installing? Are you using a specific template or just the default?

**Comment by patprache** [comment]
Thank for your reply !

I'm using the default installation on a recent linux debian.

new Service({
    name:"pv5web",
    description: "pv5web server",
    script: path.join(__dirname,"app.js")
});

The service label becomes "pvweb" somewhere in the code ... I don't know exactly where but I suspect the line:  https://github.com/coreybutler/node-linux/blob/master/lib/daemon.js#L159

I can interract with service with the command line  "$service pvweb start" for example.

**Comment by coreybutler** [comment]
Ah, the label... yeah, that does look like the cause. Have you tried correcting it on your local system?

The code  on that line should read:

```js
return this.name.replace(/[^a-zA-Z0-9\_]+/gi,'').toLowerCase()
```

You should be able to modify your code locally. If that works, either submit a PR with the change (I'll accept it) or let me know and I'll add it/release an update.

**Comment by patprache** [comment]
Perfect, the service label is correct and it works as usual with this code! And it's the same name as in node-windows module. Can you release an update ?

**Comment by coreybutler** [comment]
All set, released as `0.1.9`.
